src/services/api/search/result/search.result.service.ts

```diff
@@ -175,10 +181,15 @@ export class SearchResultService {
           );
           return undefined;
         }
-
-        // no authorization check - all spaces must be visible
-        // context, profile and others are readable by all
-
```

Comment:
@hero101, this is causing a ton of issues in the search result - for example, a callout visible to the user is matched, but one of the parent spaces is private - it is returned, and the user has no access to it, hence the search is throwing an error.